app/Http/Controllers/JournalController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Request\JournalRequest;
+
+class JournalController extends Controller
+{
+    public function create(JournalRequest $request)
```

Comment:
Change function name to `store`. `create` is used to get the form, which we don't need in API.